controller/contribution/revision.php

```diff
@@ -736,6 +744,9 @@ protected function process_steps()
 
 		$step = $steps[$step_num];
 
+		// Check if we are to skip the EPV
+		$this->skip_epv = $this->auth->acl_get('u_titania_mod_extension_validate') && (boolean) $this->request->variable('skip_epv', 0);
```

Comment:
The call to `request->variable` can just have the second parameter as `false` and then the casting isn't needed.